src/common/constants/constants.ts

```diff
@@ -103,7 +103,10 @@ export const ApiVersion = {
     armClassicStorage: "2016-11-01",
     armStorage: "2016-12-01",
     armBatch: "2019-08-01",
-    compute: "2019-03-01",
+    compute: {
+        default: "2019-03-01",
+        skus: "2019-04-01",
```

Comment:
Any reason we can't remove default and use 2019-04-01 for all compute api calls?

Reply:
Dave and I took a look at this and it seems like sometimes the APIs within the Compute API sometimes has different versioning so we did this to modularize and for more specific testing.